### PR TITLE
(feat) - O3-3222 - ward app - add patient card element to include risk factor obs within the current visi

### PR DESCRIPTION
**Pull request by chibongho**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Adds a new patient card element to display obs of coded values (typically answers from checkbox fields in a form.)

## Screenshots

image

## Other

This PR currently includes commit from [this](https://github.com/openmrs/openmrs-esm-patient-management/pull/1207/files), which hasn't made it into the main branch yet. I will rebase accordingly once that's the case.

Some patient registration test cases (unrelated to this PR) from `yarn verify` is failing due to mocking issues. This commit is pushed with `git push --no-verify`. I will reverify once that's fixed.

### CODE REVIEW

File: packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      'visit:(uuid,display)))';
+
+//  get the setMembers of a concept set
+const conceptSetCustomRepresentation = 'custom:(uuid,setMembers:(uuid))';
+
+export function useConceptToTagColorMap(codedObsConfig: PatientCodedObsElementConfig) {
+  // fetch the members of the concept sets and process the data
+  // to return conceptToTagColorMap (wrapped in a promise).
+  // Let swr cache the result of this function.
+  const fetchAndMap = (url: string) => {
```
[the remark this answers is not in the document]
Comment (mseaton) [reply packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts:19]: Defer to others on this, and best way to optimize.

**Review by mseaton — APPROVED** [review]
Generally LGTM

File: packages/esm-ward-app/src/config-schema.ts [hunk 2]
```diff
@@ -90,6 +91,48 @@ export const configSchema: ConfigSchema = {
...
+              _type: Type.UUID,
+              _description: 'Required. Defines which observation to show',
+              _default: null,
+            },
+            summaryLabel: {
+              _type: Type.String,
+              _description: `Optional. The custom label or i18n key to the translated label to display for the summary tag. The summary tag shows the count of the number of answers that are present but not configured to show as their own tags. If not provided, defaults to the name of the concept.`,
+              _default: null,
+            },
+            summaryLabelI18nModule: {
```
[the remark this answers is not in the document]
Comment (chibongho) [reply packages/esm-ward-app/src/config-schema.ts:106]: There is similar config in the service queues app as well. I filed https://openmrs.atlassian.net/browse/O3-3480 to remove those config options from here and there.

File: packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts:19]: Cool idea! Interested in what @denniskigen thinks.

[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.resource.ts:19]: Ping @denniskigen , could I get your feedback on this pattern? You know the SWR caching behavior better than anyone, as far as I'm aware.

**Pull request merged**